src/altinn-app-frontend/src/features/expressions/shared-tests/functions/not/not-erros-on-two-arguments.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Not errors zero arguments",
```

Comment:
This test seems to have wrong name

Reply:
Tagging @ivarne. I'm changing it to `Not errors two arguments` (and also renaming `erros` -> `errors` in the filename)